lib/src/platform/src/realtime/realtime.dart

```diff
@@ -105,31 +108,48 @@ class Realtime extends PlatformObject {
   //   throw UnimplementedError();
   // }
 
-  /// gets stats based on params as a [PaginatedResult]
+  /// Queries the REST `/stats` API and retrieves your application's usage
+  /// statistics. You must specify the [start] time from which stats are
+  /// retrieved, specified as milliseconds since the Unix epoch, and the [end]
+  /// time until stats are retrieved, specified as milliseconds since the Unix
+  /// epoch.
+  /// Set the [direction], which describes the order in which stats are returned
+  /// in. Valid values are `backwards` which orders stats from most recent to
+  /// oldest, or `forwards` which orders stats from oldest to most recent. The
+  /// default is `backwards`.
+  /// Provide [limit], which specifies upper limit on the number of
+  /// stats returned (the default is 100, and the maximum is 1000).
+  /// Set the [unit] as either a `minute`, `hour`, `day` or `month`. Based on
+  /// the unit selected, the given start or end times are rounded down to the
+  /// start of the relevant interval depending on the unit granularity of the
+  /// query.
+  /// Returns a [PaginatedResult] object containing an array of
+  /// [Stats] objects.
   ///
-  /// https://docs.ably.com/client-lib-development-guide/features/#RSC6
+  /// See the [Stats docs](https://ably.com/docs/general/statistics).
   // Future<PaginatedResult<Stats>> stats([Map<String, dynamic>? params]) {
```

Comment:
This method does not match the docstring and is commented out. Shouldn't we remove it?

Reply:
I'll remove both this one and the `request` method above, especially since mentioning the non-existing properties (like [limit], [unit] etc.) causes warnings, and that in turn causes CI to fail.

Done in 7d70f4773293bae5d0d57b1c19b134346e9096f9